test/index.spec.ts

```diff
@@ -1,3 +1,5 @@
+// Do we want logging in the tests?
+/* eslint-disable no-console */
```

Comment:
Not sure if we care about `console.*` statements in the tests.

Personally, I'm not sure we should be including additional logging in the tests as the tests should provide the information we need.

Thoughts?

Reply:
I don't think they're necessary in the tests either.